stream.go

```diff
@@ -50,9 +50,15 @@ func Subscribe(url, lastEventId string) (*Stream, error) {
 
 // SubscribeWithRequest will take an http.Request to setup the stream, allowing custom headers
 // to be specified, authentication to be configured, etc.
-func SubscribeWithRequest(lastEventId string, req *http.Request) (*Stream, error) {
+func SubscribeWithRequest(lastEventId string, request *http.Request) (*Stream, error) {
+	return SubscribeWith(lastEventId, http.DefaultClient, request)
+}
+
+// SubscribeWith subscribes to the Events emitted from the specified url
```

Comment:
Great, probably need to specify the obvious and say that SubscribeWith takes a request and a client to allow custom TLS, or something explanatory like that. If you do that, then either squash the commits or open a new PR, I'll merge :-)